document_sftp/models/document_sftp_root.py

```diff
@@ -59,6 +63,18 @@ def _open(self, path, flags, attr):
         raise NotImplementedError()
 
     @api.model
+    def _remove(self, path):
+        """Return file attributes"""
+        raise NotImplementedError()
```

Comment:
Is it recommended to return an error at this level? Shouldn't we return a forbidden code or similar?